src/query/query_executor.c

```diff
@@ -23872,6 +23877,16 @@ qexec_evaluate_aggregates_optimize (THREAD_ENTRY * thread_p, AGGREGATE_TYPE * ag
 		  error = er_errid ();
 		  return (error == NO_ERROR ? ER_FAILED : error);
 		}
+
+	    }
+
+	  /* agg_index_name is set to show it in the trace information */
+	  spec->s_id.scan_stats.agg_index_name = NULL;
+	  error = heap_get_indexinfo_of_btid (thread_p, &ACCESS_SPEC_CLS_OID (spec), &agg_ptr->btid,
+					      NULL, NULL, NULL, NULL, &spec->s_id.scan_stats.agg_index_name, NULL);
+	  if (error != NO_ERROR)
+	    {
+	      return error;
 	    }
```

Comment:
This is unnecessary if no trace is output.
```suggestion
	  if (thread_is_on_trace (thread_p))
	    {
	      /* agg_index_name is set to show it in the trace information */
	      spec->s_id.scan_stats.agg_index_name = NULL;
	      error = heap_get_indexinfo_of_btid (thread_p, &ACCESS_SPEC_CLS_OID (spec), &agg_ptr->btid,
						  NULL, NULL, NULL, NULL, &spec->s_id.scan_stats.agg_index_name, NULL);
	      if (error != NO_ERROR)
		{
		  return error;
		}
	    }
```

Reply:
OK, I aggree.